_data/simple-icons.json

```diff
@@ -10226,17 +10232,17 @@
         {
             "title": "X.Org",
             "hex": "F28834",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/9/90/X.Org_Logo.svg"
+            "source": "https://en.wikipedia.org/wiki/File:X.Org_Logo.svg"
         },
         {
             "title": "Xamarin",
             "hex": "3498DB",
-            "source": "https://github.com/dotnet/swag/tree/master/xamarin"
+            "source": "https://github.com/dotnet-foundation/swag/blob/main/xamarin/Xamarin/svg/Xamarin%20logo.svg"
```

Comment:
Should be https://github.com/dotnet-foundation/swag/blob/7d5581d62e22668315de27303e4f714fac9ab1c1/xamarin/Xamarin/svg/Xamarin%20logo.svg